README.md

```diff
@@ -70,7 +74,199 @@ bundle install
 
 ## Configuration
 
-### 1. Configure BigQuery connection, feature flags etc
+
+### 1. Get a BigQuery project setup and add initial owners
+
+Ask in Slack on the `#twd_data_insights` channel for someone to help you
+procure a BigQuery instance in the `digital.education.gov.uk` Google Cloud
+Organisation.
+
+Ask for your `@digital.education.gov.uk` Google account to be setup as an owner
+via the IAM and Admin settings. Add other team members as necessary.
+
+#### Set up billing
+
+You also need to set up your BigQuery instance with paid billing. This is
+because `dfe-analytics` uses streaming, and streaming isn't allowed in the free
+tier:
+
+```bash
+accessDenied: Access Denied: BigQuery BigQuery: Streaming insert is not allowed
+in the free tier
+```
+
```

Comment:
Might be worth adding that typically the delivery manager can help arrange this, although we can't include the name of who to email here, we should have it in some other documentation somewhere that's not publicly accessible.